guacamole/src/main/java/org/glyptodon/guacamole/net/basic/rest/ObjectRetrievalService.java

```diff
@@ -48,7 +49,7 @@
      * @param session
      *     The GuacamoleSession to retrieve the UserContext from.
      *
-     * @param identifier
+     * @param authProviderIdentifier
      *     The unique identifier of the AuthenticationProvider that created the
      *     UserContext being retrieved. Only one UserContext per
```

Comment:
This is a little hard to understand. One UserContext per AuthenticationProvider? Surely that's per user?